core/os/android/adb/commands.go

```diff
@@ -40,7 +40,7 @@ const (
 	ErrRootFailed      = fault.Const("Device failed to switch to root")
 
 	maxRootAttempts                            = 5
-	gpuRenderingStagesDataSourceDescriptorName = "gpu.renderingstages"
+	gpuRenderingStagesDataSourceDescriptorName = "gpu.renderstages"
```

Comment:
I guess we should change the constant to `gpuRenderStagesDataSourceDescriptorName`

Reply:
Done